lib/media/track/videotrack.js

```diff
@@ -83,21 +86,20 @@ class VideoTrack extends MediaTrack {
   _checkIfCanCaptureFrames(isPublishing = false) {
     let canCaptureFrames = true;
     let message = '';
-    const { enabled, readyState } = this.mediaStreamTrack;
 
-    if (!enabled) {
+    if (!this.mediaStreamTrack) {
```

Comment:
In what scenarios `mediaStreamTrack` would be null and `_checkIfCanCaptureFrames` is called?

Reply:
In Large Rooms, there won't be a MediaStreamTrack associated with a RemoteTrack that is switched off. In that case, `_checkIfCanCaptureFrames()` should return false.